src/octoprint/plugins/virtual_printer/__init__.py

```diff
@@ -55,6 +55,10 @@ def get_settings_defaults(self):
                 "EMERGENCY_PARSER": True,
                 "EXTENDED_M20": False,
             },
+            "m115ReportExtraParams": True,
+            "extraParams": {
+                "area": "{full:{min:{x:0.00,y:0.00,z:0.00},max:{x:200.00,y:200.00,z:200.00}},work:{min:{x:0.00,y:0.00,z:0.00},max:{x:200.00,y:200.00,z:200.00}}}",
+            },
```

Comment:
From what I understand, this is tied to a capability (`M115_GEOMETRY_REPORT`), so don't make it `extraParams` or something like that but rather tie it to the configured capabilities here as well.

Also, this should return more realistic parameters, 10x10x10cm buildvolumes are rather uncommon I'd say. I suggest going with the values from the default printer profile here. The virtual printer could even read that from the global config and return it here to be consistent. Ideally this should use place holders and be considered a format string for that matter.